setup.py

```diff
@@ -56,7 +54,9 @@
         "openvino": ["openvino-dev==2022.1.0"],
         "docs": ["numpydoc"],
         "tf": [
-            "tensorflow>=2.0,<=2.10"
+            "tensorflow>=2.0,<=2.10",
+            "tensorpack>=0.11",
+            "tf_slim>=1.1.0",
         ],  # Last supported TF version on Windows Native is 2.10
         "apple_mchips": ["tensorflow-macos<2.13.0", "tensorflow-metal"],
```

Comment:
I believe we also need to add `tf_slim` and `tensorpack` for `apple_mchips`:

```suggestion
        "apple_mchips": [
            "tensorflow-macos<2.13.0",
            "tensorflow-metal",
            "tensorpack>=0.11",
            "tf_slim>=1.1.0",
        ],
```